source/sgp_mode/HealthHost.h

```diff
@@ -83,6 +117,21 @@ class HealthHost : public SGPHost {
               sym_cycle = 0;
             }
           }
+          }
+        }
+
+        if(sym_cycle == 0){
```

Comment:
Some documentation explaining what this sequence is for and what it is currently doing would be good